packages/manager/modules/sms/src/sms/sms.routing.js

```diff
@@ -0,0 +1,42 @@
+import { FeatureAvailability } from '@ovh-ux/ng-ovh-telecom-universe-components';
+import { SMS_AVAILABILITY } from '../feature-availability/feature-availability.constants';
+
+import controller from './telecom-sms.controller';
+import smsView from './telecom-sms.html';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('sms.service', {
+    url: '/:serviceName',
+    views: {
+      '': {
+        template: smsView,
+        controller,
+        controllerAs: 'TelecomSmsCtrl',
+      },
+    },
+    abstract: true,
+    resolve: {
+      initSms: ($q, $stateParams, TucSmsMediator) => {
+        // init sms services
+        TucSmsMediator.initAll().then((smsDetails) => TucSmsMediator
```

Comment:
You don't chain ? To make sure stufss are initialized before saying they are

Reply:
Actually I just extracted the existing routing, some rework might be needed in the future 👀